src/Features/CSharpTest/Microsoft.CodeAnalysis.CSharp.Features.UnitTests.csproj

```diff
@@ -16,4 +16,5 @@
   <ItemGroup>
     <PackageReference Include="Basic.Reference.Assemblies.Net60" Version="$(BasicReferenceAssembliesNet60Version)" />
   </ItemGroup>
+  <Import Project="..\..\Analyzers\CSharp\Tests\CSharpAnalyzers.UnitTests.projitems" Label="Shared" />
```

Comment:
this is major.  all the C# code style tests *all* run at the feature layer, not hte editor feature layer.